cogment_verse/specs/data.proto

```diff
@@ -58,6 +58,10 @@ message Observation {
   optional nd_array.Array action_mask = 3;
   optional bytes rendered_frame = 4;
   repeated string overridden_players = 5;  // list of players that provided an action that was overriden during the last tick
+  optional string game_player_name = 6;
```

Comment:
This shouldn't be needed, it's accessible using the specs.

Reply:
This field will change for every timestep